specification/web/resource-manager/Microsoft.Web/stable/2021-01-01/KubeEnvironments.json

```diff
@@ -0,0 +1,650 @@
+            "description": "Name of the Kubernetes Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "kubeEnvironmentEnvelope",
+            "in": "body",
+            "description": "Configuration details of the Kubernetes Environment.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/KubeEnvironment"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/KubeEnvironment"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/KubeEnvironment"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Create kube environments": {
+            "$ref": "./examples/KubeEnvironments_CreateOrUpdate.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "KubeEnvironments"
+        ],
+        "summary": "Delete a Kubernetes Environment.",
+        "description": "Description for Delete a Kubernetes Environment.",
+        "operationId": "KubeEnvironments_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the Kubernetes Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Delete operation completed"
+          },
+          "202": {
+            "description": "Delete operation is in progress"
+          },
+          "204": {
+            "description": "Kubernetes Environment does not exist"
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete kube environment by name": {
+            "$ref": "./examples/KubeEnvironments_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
```

Comment:
>"patch" [](http://example.com/codeflow?start=6&length=7)

Your patch operation is not async right? just making sure.

Reply:
Adding @vinisoto to help here as @JennyLawrance is OOF this afternoon

Reply:
Correct. Not async.